src/configuration.ts

```diff
@@ -45,6 +51,51 @@ export const DEFAULT_FILTER_DISTANCE_CONFIG: FilterDistanceConfig = {
 }
 
 export class Configuration {
+    // Values useful for generating HTML for tool
```

Comment:
technically all of these can be overwritten via the constructor, but i dont feel like listing em all in the api_spec,,,